pkg/agent/agent.go

```diff
@@ -85,6 +92,8 @@ type NimbessAgent struct {
 	Pipelines map[string]*NimbessPipeline
 	// MetaPipelines contains a map of network name to abstract pipelines
 	MetaPipelines map[string]*NimbessPipeline
+    l2fib map[string]L2FIBEntry
+    notifications chan network.L2FIBCommand
```

Comment:
in the future, we may want to make this chan of a more generic type. Then we can process multiple different types of notifications on the same channel. What do you think?

Reply:
That means using reflection.*. I tried that, the recovery a complex notification via reflection is beyond ugly.